app/controllers/internal/response_templates_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Internal::ResponseTemplatesController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @response_templates = if params[:filter]
+                            ResponseTemplate.where(type_of: params[:filter])
+                          else
+                            ResponseTemplate.all
+                          end
+    @response_templates = @response_templates.page(params[:page]).per(50)
+  end
+
+  def create
+    @response_template = ResponseTemplate.new(permitted_params)
+    if @response_template.save
+      flash[:success] = "Response Template: \"#{@response_template.title}\" saved successfully."
+      redirect_to("/internal/response_templates/#{@response_template.id}/edit")
```

Comment:
@Zhao-Andy I'm currently on a mission to move us towards using the URL helpers wherever we can vs manually constructing URLs:

```ruby
redirect_to edit_internal_response_template(@response_template)
```